.travis.yml

```diff
@@ -10,18 +10,19 @@ python:
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
+# Use the test toolchain repository to get the latest GCC.
 before_install:
     - sudo apt-get update
     - sudo apt-get install -y fitsverify
 
 install:
     - pip install -U pip
     - pip install twine
-    - for i in $(ls -d */); do cd $i; pip install -r dev_requirements.txt; cd ..; done
+    - for i in cadcutils cadcdata cadctap cadcetrans cadccutout; do cd $i; pip install -r dev_requirements.txt; cd ..; done
```

Comment:
I had a `docker` folder in there for ease of testing and development, which was getting in the way of Travis.  I forced it to just look at the `cadc*` folders, but I've since removed the `docker` folder, so I restored this file to the mainline version.